frontend/src/components/widgets/inputs/FontInput.vue

```diff
@@ -125,50 +162,90 @@ export default defineComponent({
 				this.$emit("update:fontFamily", newName);
 
 				fontFamily = newName;
-				fontStyle = (await this.fonts.getFontStyles(newName))[0];
+				fontStyle = "Normal (400)";
```

Comment:
It's preferable if we can define this only in that one location of the constants file in the backend, rather than spread across the frontend and backend. What necessitated the change here? (I admittedly haven't looked very closely at what the old line was and why that old line doesn't work anymore and had to be changed to this.)

Reply:
Previously, when you selected a new font, then it would select the first style (often a light one). This changes it so it loads the normal weight when you select a new font.

Reply:
Ah, I see, that's helpful.